.github/workflows/mac-wpt.yml

```diff
@@ -30,18 +30,18 @@ jobs:
       matrix:
         chunk_id: [1, 2, 3, 4, 5]
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         if: github.event_name != 'pull_request_target'
         with:
           fetch-depth: 2
       # This is necessary to checkout the pull request if this run was triggered via a
       # `pull_request_target` event.
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         if: github.event_name == 'pull_request_target'
         with:
           ref: refs/pull/${{ github.event.number }}/head
           fetch-depth: 2
-      - uses: actions/download-artifact@v3
```

Comment:
mac workflow still uses upload-artifact@v3 and you are not allowed to mix v3/v4 for upload download as they are incompatible.